IFC4x3/Properties/n/NumberOfPhases_0ZOkcOH$14AAoaednCd9ms/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfPhases_0ZOkcOH14AAoaednCd9ms" Name="NumberOfPhases" UniqueId="2362e998-47f0-4428-aca4-a27c4c9c9c36" PrimaryDataType="IfcInteger" />
```

Comment:
inconsistancy across PRs PhaseNumber is used on other requests please review and update accordingly

Reply:
Now Ok, I reused "NumberOfPhases_1BIplyxBLFfuml0KvHZ1y2". Commit :  840d7b293d3051eca8050639884528e00dfc5d4e